_maps/map_files/EchoStation/map_adjustment_echo.dm

```diff
@@ -0,0 +1,26 @@
+/*
+			< ATTENTION >
+	If you need to add more map_adjustment, check 'map_adjustment_include.dm'
+	These 'map_adjustment.dm' files shouldn't be included in 'dme'
+*/
+
+/datum/map_adjustment/EchoStation
+	map_file_name = "EchoStation.dmm"
+	no_lavaland = TRUE
+	central_orbit = /datum/orbital_object/z_linked/station
```

Comment:
No lavaland, the mine type and the orbital center should be part of the json config. This datum should do code changes only like the code changes needed for job adjustments